app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -186,21 +188,25 @@ export const ListingDetailsEligibility = ({
           <StandardTable headers={occupancyTableHeaders} data={occupancyTableData} />
         </ListSection>
 
-        <ListSection
-          title={t("listings.lottery.title")}
-          subtitle={
-            <>
-              <div className="mb-4">
-                {renderInlineMarkup(t("listingsForSale.lotteryPreferences.noPreferences"))}
-              </div>
-              {t("listingsForSale.lotteryPreferences.hasPreferences")}
-            </>
-          }
-        >
-          <ErrorBoundary boundaryScope={BoundaryScope.component}>
-            <ListingDetailsPreferences listingID={listing.listingID} />
-          </ErrorBoundary>
-        </ListSection>
+        {isEducator(listing) ? (
+          <ListingDetailsChisholmPreferences />
+        ) : (
+          <ListSection
+            title={t("listings.lottery.title")}
+            subtitle={
+              <>
+                <div className="mb-4">
+                  {renderInlineMarkup(t("listingsForSale.lotteryPreferences.noPreferences"))}
+                </div>
+                {t("listingsForSale.lotteryPreferences.hasPreferences")}
+              </>
+            }
+          >
+            <ErrorBoundary boundaryScope={BoundaryScope.component}>
+              <ListingDetailsPreferences listingID={listing.listingID} />
+            </ErrorBoundary>
```

Comment:
I noticed that there is existing usage of the `ErrorBoundary` component. Do we need to use that in the `ListingDetailsChisholmPreferences` component?